.buildkite/scripts/package.sh

```diff
@@ -8,6 +8,7 @@ source ./bin/activate-hermit
 CLOUDBEAT_VERSION=$(grep defaultBeatVersion version/version.go | cut -d'=' -f2 | tr -d '" ')
 PYTHON_BIN=./build/ve/$(go env GOOS)/bin
 PYTHON=$PYTHON_BIN/python
+VERSION_QUALIFIER="${VERSION_QUALIFIER:-""}"
```

Comment:
I'm trying to understand why here you used `:-` and on publish.sh `:=`

Reading https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html I think we should be using `:-` on both for substitution in case of not available?

Reply:
Essentially, there is no difference between the two since, in both cases, we assign the outcome of the expansion back to the variable. 

We would have different result if we didn't assign the expansion back to the variable.


I will make both the same for consistency.